templates/jail.local.j2

```diff
@@ -1,3 +1,9 @@
+{% if ansible_distribution_major_version | int >= 12 %}
+[DEFAULT]
+backend = systemd
+logtarget = SYSTEMD-JOURNAL
```

Comment:
`logtarget = SYSTEMD-JOURNAL` belongs to `fail2ban.conf` not to jail config

Reply:
I moved `logtarget = SYSTEMD-JOURNAL` configuration to `/etc/fail2ban/fail2ban.local`, see 0b9a774f875ff3c175fc4f16a6d2b5b479d17e45 templates/fail2ban.local.j2